test/migrations/namespace_get_function.hrx

```diff
@@ -0,0 +1,39 @@
+<==> input/entrypoint.scss
+@import "library";
+
+$fn1: get-function(increment);
+$fn2: get-function($name: "increment");
+$fn3: get-function("incre" + "ment");
+
+$x: "increment";
+$fn4: get-function($x);
+
+<==> input/_library.scss
+@function increment($x) {
+  @return $x + 1;
+}
+
+<==> output/entrypoint.scss
+@use "sass:meta";
+@use "library";
+
+$fn1: meta.get-function(library.increment);
```

Comment:
I don't think it matters given the new module syntax, but FYI `library.increment` isn't a valid SassScript expression. `.` isn't a valid identifier character.